backend/molgenis-emx2-tasks/src/main/java/org/molgenis/emx2/tasks/Task.java

```diff
@@ -284,6 +289,15 @@ public String getCronExpression() {
     return this.cronExpression;
   }
 
+  public T cronUserName(String cronUserName) {
+    this.cronUserName = cronUserName;
```

Comment:
this is not used right? it is now 'admin only'?

Reply:
yes bit its used in code, only partially loading taks state from db into object makes it hard keep the logic working , as this issue shows.